daemon/network.go

```diff
@@ -496,6 +543,28 @@ func (daemon *Daemon) DeleteNetwork(networkID string) error {
 	return daemon.deleteNetwork(networkID, false)
 }
 
+func (daemon *Daemon) deleteLoadBalancerSandbox(n libnetwork.Network) {
+
```

Comment:
I don't see a disableService here, wondering why there is an asymmetry between the EnableService in the Create and no need for the disableService here

Reply:
Oops. I think I missed this in the last PR.  I'll update it.  sorry.

Reply:
Address in the latest PR.